layouts/_default/single.html

```diff
@@ -71,7 +71,13 @@ <h1>{{ .Page.Title }}</h1>
         <!--- Improve this page button --->
         {{ if site.Params.GitRepo }}
           <div class="btn-improve-page">
-              <a href="{{ site.Params.GitRepo }}/edit/{{ site.Params.GitBranch }}/content/{{ .File.Path }}" title="{{ i18n "improve_this_page" }}" target="_blank" rel="noopener">
+            {{ if ( eq site.Params.GitForge "gitlab" ) }}
+              <a href="{{ site.Params.GitRepo }}-/edit/{{ site.Params.GitBranch }}/{{ .File.Path }}" title="{{ i18n "improve_this_page" }}" target="_blank" rel="noopener">
```

Comment:
Should it be `"{{ site.Params.GitRepo }}/-/edit/` instead of `"{{ site.Params.GitRepo }}-/edit/` ?

Reply:
Hum I see, I believe the problem is my GitRepo param is styled as https://codeberg.org/FlyingFlamingo/pablomarcos.me/ instead of  https://codeberg.org/FlyingFlamingo/pablomarcos.me, so the {{ site.Params.GitRepo }} makes the / redundant. I guess for backwards compatibility I should add this change, even though I dont know which is more common, to have the final / or not